nemoguardrails/utils.py

```diff
@@ -302,3 +303,52 @@ def snake_to_camelcase(name: str) -> str:
         str: The converted CamelCase string.
     """
     return "".join(n.capitalize() for n in name.split("_"))
+
+
+def get_railsignore_path() -> Path:
```

Comment:
how about implementing following signature?

```python
def get_all_railsignore_paths(
    full_path: Optional[str] = None, base_dir: Optional[str] = None
) -> List[Path]:
    """Helper to get all .railsignore paths from the current directory up to the root.

    Args:
        full_path (str, optional): The full path to start from.
        base_dir (str, optional): The base directory to consider as the root.

    Returns:
        List[Path]: A list of .railsignore file paths, ordered from root to current directory.
    """
```

Reply:
Is the intention here to change the behavior such that there will potentially be multiple .railsignore files scattered across the repo rather than one .railsignore at the root directory?

Or are we simply allowing flexibility for if the .railsignore changed?